### PR TITLE
update indexes.cypher

### PR DESCRIPTION
**Pull request by sagarnavale**

### CODE REVIEW

File: cartography/data/jobs/cleanup/github_repos_cleanup.json [hunk 1]
```diff
@@ -1,42 +1,42 @@
 {
   "statements": [
     {
-      "query": "MATCH (:CloudanixWorkspace{id: $WORKSPACE_ID})-[:OWNER1]->(:GitHubOrganization)-[:OWNER]-> (n:GitHubRepository) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "query": "MATCH (:CloudanixWorkspace{id: $WORKSPACE_ID})-[:OWNER1]->(:GitHubOrganization{id:$Organization_id})-[:OWNER]-> (n:GitHubRepository) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```
Comment (ganeshjagdale9545) [inline cartography/data/jobs/cleanup/github_repos_cleanup.json:4]: rename Organization_id to ORGANIZATION_ID

File: cartography/intel/github/__init__.py [hunk 2]
```diff
@@ -34,27 +42,57 @@ def start_github_ingestion(neo4j_session: neo4j.Session, config: Config) -> None
     # run sync for the provided github tokens
     for auth_data in auth_tokens['organization']:
         try:
+            user_data, org_data = cartography.intel.github.users.get( auth_data['token'],auth_data['url'], auth_data['name'])
+            common_job_parameters['Organization_id']=org_data.get('url')
```
Comment (ganeshjagdale9545) [inline cartography/intel/github/__init__.py:46]: use capital case for this

**Review by ganeshjagdale9545 — APPROVED** [review]
looks good

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**